src/utils/formate-axios-error.ts

```diff
@@ -0,0 +1,9 @@
+import { AxiosError } from "axios"
+
+const formateAxiosError = (err: unknown) => {
+    if (err instanceof Error) return err
+    else if (err instanceof AxiosError) return new Error(AxiosError.name)
+    else return new Error('Unknown Error')
+}
```

Comment:
[+] Здесь и в похожих местах более плоских код легче читается. И правилом хорошего тона является проставление в конце точек с запятой))

```suggestion
const formateAxiosError = (err: unknown) => {
    if (err instanceof Error) {
        return err;
    }

    if (err instanceof AxiosError) {
        return new Error(AxiosError.name);
    }

    return new Error('Unknown Error');
}
```